test/unittest/ext/utils.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from superduperdb.ext.utils import format_prompt, get_key
+
+
+def test_patch_environment_variable(monkeypatch):
```

Comment:
One might even question the need for this unit test :-D but I will never turn down a unit test.